Add documentation for GCP Service Classes (#1717)
kyma-project/kyma: docs/service-brokers/gcp-broker-service-classes/bigquery/docs/overview.md

---
title: Overview
type: Overview
---

Google **BigQuery** is a serverless, highly scalable enterprise data warehouse designed to make all your data analysts productive at an unmatched price-performance.

> BigQuery allows you to capture and analyze data in real time using powerful streaming ingestion capability so that your insights are always current.

Reviewer: ```suggestion
BigQuery allows you to capture and analyze data in real time using powerful streaming ingestion capability so that your insights are always up to date.
```

Small thing - maybe `up to date` sounds better.

For more details, read the **Plans Details** document.

> >**NOTE:** Google does not provide the ability to specify the broker's version so bear in mind that you use the most up-to-date version of the GCP Broker.

Reviewer: ```suggestion
>**NOTE:** Google cannot specify the broker's version so bear in mind that you use the most up-to-date version of the GCP Broker.
```
Just a suggestions to make it shorter.
